### ISSUE TITLE
Catalogue Name in Unicode display problem

### ISSUE THREAD

**Issue by beetaa** [Gorgious56/asset_browser_utilities#140]
blender's catalogue definition file is encode in Unicode, but catalogue name in non-English cannot parsed normaly by asset_browser_utilities addon. snapshot as below:

![image](https://github.com/Gorgious56/asset_browser_utilities/assets/5688224/f965bc6e-7f0b-4a28-bd0d-97e34f55577a)

**Comment by Gorgious56** [comment]
Hello, thanks for the report. Can you share the blender-assets.cats.txt
that is at your root asset library folder so I can debug this ? This is a
known issue with enum properties. Thank you.

**Comment by beetaa** [comment]
@Gorgious56 Yes, the file is as attachment, Thanks for your nice job and shares.

[blender_assets.cats.txt](https://github.com/Gorgious56/asset_browser_utilities/files/14965910/blender_assets.cats.txt)

**Comment by beetaa** [comment]
@Gorgious56 hi, I think you can see the below two issues from other repo:

https://github.com/johnnygizmo/BulkAsset/issues/2
https://github.com/johnnygizmo/BulkAsset/issues/4

the second one give a solution and code for the same issue.

**Comment by Gorgious56** [comment]
FWIw it's due to this known bug in the Blender API https://blender.stackexchange.com/questions/216230/is-there-a-workaround-for-the-known-bug-in-dynamic-enumproperty 
Cheers
